### ISSUE TITLE
2020-12-25

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#404]
461artificial intelligence and statistics 2010, cited by 10926; 
462Abstract
463superiority of deeper vs less deep architectures; why standard gradient descent from random initialization is doing so poorly with deep neural networks; first observe non-linear activations functions; logistic sigmoid activation is unsuited for deep networks with random initialization; propose a new initialization scheme that brings substantially faster convergence; (2020-12-14)
464I.Deep Neural Networks
4651.learning feature hierarchies with features from higher levels of the hierarchy formed by the composition of lower level features; learn the kind of complicated functions that can represent high-level abstractions; (2020-12-14)
4662.initialization or training schemes different from the classical feedforward neural networks; better than the standard random initialization and gradient-based optimization; analyzing what may be going wrong with good old (but deep) multilayer neural networks; (2020-12-14)
4673.Monitor activations and gradients; evaluate initialization procedure; (2020-12-15)
468II.Experimental Setting and Datasets
469II.1 Online Learning on an Infinite Dataset: Shapeset-3 × 2
